modal/experimental/flash.py

```diff
@@ -616,3 +643,108 @@ async def flash_get_containers(app_name: str, cls_name: str) -> list[dict[str, A
     req = api_pb2.FlashContainerListRequest(function_id=fn.object_id)
     resp = await client.stub.FlashContainerList(req)
     return resp.containers
+
+
+def _http_server(
+    port: int,
+    *,
+    proxy_region: Literal[
+        "us-east", "us-west", "ap-south", "True"
+    ],  # Not defaulting since changing user behavior after allowing a default is tricky
+    startup_timeout: Optional[int] = None,
+    exit_grace_period: Optional[int] = None,
+):
+    """Decorator for Flash-enabled HTTP servers on Modal classes.
+
+    Args:
+        port: The local port to forward to the Flash service.
+        proxy_region: The region to proxy the Flash service to.
+        startup_timeout: The maximum time to wait for the Flash service to start.
+        exit_grace_period: The time to wait for the Flash service to exit gracefully.
```

Comment:
Is the "Flash service" the thing that's starting and stopping if you're using this on a "normal" web server function?